code/game/objects/items/eightball.dm

```diff
@@ -137,38 +169,31 @@
 		if(isobserver(usr))
 			interact(usr)
 
-/obj/item/toy/eightball/haunted/proc/get_vote_tallies()
-	var/list/answers = list()
-	for(var/ckey in votes)
-		var/selected = votes[ckey]
-		if(selected in answers)
-			answers[selected]++
-		else
-			answers[selected] = 1
-
-	return answers
+/obj/item/toy/eightball/haunted/get_answer()
+	var/top_amount = 0
+	var/top_vote
 
+	for(var/vote in votes)
+		var/amount_of_votes = length(votes[vote])
+		if(amount_of_votes > top_amount)
+			top_vote = vote
+			top_amount = amount_of_votes
+		//If one option actually has votes and there's a tie, pick between them 50/50
+		else if(top_amount && amount_of_votes == top_amount && prob(50))
```

Comment:
In a 3 way tie, the "no" option will have twice the chance of being picked due to only having to win the prob once. Its not super important.

Reply:
oh shidd, I doubt a three way tie will ever happen

Reply:
It will, but its whatever. I cant think of any solution that isnt overengineered for a haunted eightball.